docs/language/operators/top.md

```diff
@@ -0,0 +1,40 @@
+### Operator
+
+&emsp; **top** &mdash; get top N sorted values of input sequence
+
+### Synopsis
+
+```
+top <uint> <expr> [, <expr> ...]
+```
+### Description
+
+The `top` operator returns the top N values from a sequence sorted in descending
+order by one or more expressions. `top` is functionally similar to
```

Comment:
`<uint>` isn't right since the first argument isn't an unsigned integer but rather a compile-time constant expressions that evaluates to a positive integer. For example, this works.

```
const limit = 5
top limit this
```

Maybe say something like this (inspired by the synopsis in docs/language/operators/switch.md)?

```suggestion
top <const-expr> <expr> [, <expr> ...]
```
### Description

The `top` operator returns the top N values from a sequence sorted in descending
order by one or more expressions. N is given by `<const-expr`, a compile-time
constant expression evaluating to a positive integer.

`top` is functionally similar to
```